quickwit-cli/src/index.rs

```diff
@@ -34,12 +34,15 @@ use quickwit_common::GREEN_COLOR;
 use quickwit_config::{
```

Comment:
Essentially modifications due to the move of code from quickwit-core to quickwit-control-plane.